java/src/jmri/jmrix/can/cbus/swing/modules/AbstractEditNVPane.java

```diff
@@ -54,4 +54,9 @@ private void setNV(CbusNode node) {
      * @param e the change event
      */
     abstract public void tableChanged(TableModelEvent e);
+
+    @Override
+    public void dispose() {
+        super.dispose();
+    }
```

Comment:
Why is this needed? If the method `dispose()` does not exists in this class, the parent method will be called.